python/ray/worker.py

```diff
@@ -396,6 +396,7 @@ def sigterm_handler(signum, frame):
         sys.exit(0)
 
 
+@client_mode_hook
 def get_gpu_ids():
```

Comment:
In a previous commit, was missing this line. (This is the main point of this PR.)